### PR TITLE
cell component was added; edit translation UI added

### PR DESCRIPTION
**Pull request by pavel-kru**
@Pavel-Yarakhovich

### CODE REVIEW

File: src/components/csvLocalizer/jsonLocalizer.js [hunk 1]
```diff
@@ -51,20 +76,44 @@ const JsonLocalizer = () => {
...
+                    {values.map((v, i) => (
+                      <Cell
+                        key={`${key}-${i}`}
+                        id={`${key}-${i}`}
+                        style={{ width: `${cellWidth}%` }}
+                        value={v}
+                        onClick={(id) => cellOnClick(id)}
+                        shownInput={shownInput}
+                        className={
+                          shownInput === `${key}-${i}` ? 'active-cell' : ''
```
Comment (Pavel-Yarakhovich) [inline src/components/csvLocalizer/jsonLocalizer.js]: ${key}-${i} - is repeated several times. Can we create a variable 'id' let's say?

File: src/components/csvLocalizer/jsonLocalizer.js [hunk 2]
```diff
@@ -51,20 +76,44 @@ const JsonLocalizer = () => {
...
+                        }
+                      />
+                    ))}
+                  </div>
+                  {shownInput?.split('-')[0] === key && (
```
Comment (Pavel-Yarakhovich) [inline src/components/csvLocalizer/jsonLocalizer.js]: Please, leave a comment about what parts does a `shownInput` consist of.

File: src/components/csvLocalizer/jsonLocalizer.js [hunk 3]
```diff
@@ -30,19 +39,35 @@ const JsonLocalizer = () => {
...
+    inputRef?.current?.focus();
+  }, []);
+
+  const onCancel = React.useCallback(() => {
+    setShownInput('');
+    setNewtranslation('');
+  }, []);
+
+  const saveTranslation = React.useCallback(() => {
+    console.log(`id: ${shownInput}`, `translation: ${newTranslation}`);
```
Comment (Pavel-Yarakhovich) [inline src/components/csvLocalizer/jsonLocalizer.js]: Is it temporary?

**Review by Pavel-Yarakhovich — CHANGES_REQUESTED** [review]

File: src/components/csvLocalizer/jsonLocalizer.js (hunk 1 above)
Comment (pavel-kru) [reply src/components/csvLocalizer/jsonLocalizer.js]: Sure.

File: src/components/csvLocalizer/jsonLocalizer.js (hunk 2 above)
Comment (pavel-kru) [reply src/components/csvLocalizer/jsonLocalizer.js]: It consists of 'row' and 'i'. In other words, this is id.  I will rename it to activeCellId

File: src/components/csvLocalizer/jsonLocalizer.js (hunk 3 above)
Comment (pavel-kru) [reply src/components/csvLocalizer/jsonLocalizer.js]: Yes, just to see we have everything that we need.

**Pull request merged**